docs/configuration.md

```diff
@@ -55,6 +55,13 @@ Lambda; https://aws.amazon.com/about-aws/whats-new/2018/10/aws-lambda-supports-f
 
 [^6]: Dirigent specific
 
+[^7] Because Knative's minimum autoscaling stable window is 6s, the minimum keep-alive for a function is 6s. This means
+that we need multiple functions to achieve RPS=1, each scaling up/and down with a 1-second delay from each other. In RPS
+mode, the number of functions for the cold start experiment is determined by the `RpsCooldownSeconds` parameter, which
+is the minimum keep-alive. Due to the implementation complexity, the cold start experiment sleeps for the first
```

Comment:
Same as in documentation in the comment before the functions.

Reply:
Fixed.